README.md

```diff
@@ -16,16 +16,16 @@ Go is one of the languages that is known for making programming fun again, and w
 
 We will be writing a Pac Man clone for the terminal. While writing a game you are exposed to several interesting requirements that make a very good ground for exploring many features of the language, including input, output, functions, pointers, concurrency and a bit of math.
 
-You also learn a bit more about the terminal and it's magical escape sequences.
+You also learn a bit more about the terminal and its magical escape sequences.
 
 ### Conference Talks
 
 If you want to have a look at the tutorial in a talk format before trying (about 25 minutes), try one of the links below:
 
 1. [Google Cloud Next UK '19](https://cloud.withgoogle.com/next/uk/sessions?session=DZ224), London, UK (November, 21st 2019)
-1. [London Gophers](https://youtu.be/SM8LTMnB4x0), London, UK
-1. [GoWayFest 3.0](https://youtu.be/0qvW4kIlS8I), Minsk, Belarus
-1. [GothamGo 2019](https://youtu.be/GH0DlCKTppE), New York City, NY, USA
+2. [London Gophers](https://youtu.be/SM8LTMnB4x0), London, UK
```

Comment:
The numbering with only 1.'s is intentional. Markdown automatically numbers it correctly for us. Please revert.

Reply:
Alrighty. I know that Markdown numbers it correctly. It's just that it doesn't look friendly in the source. I'll revert it then ;)

Reply:
That's me being lazy: when I add a new talk I don't want to renumber everything (they are in reverse chronological order). :)